include/lsst/jointcal/AstrometryModel.h

```diff
@@ -61,6 +62,9 @@ class AstrometryModel {
 
     virtual ~AstrometryModel(){};
 
+    /// Return true if this is a "reasonable" model.
+    bool validate(CcdImageList const &ccdImageList) const { return true; }
```

Comment:
Thanks, I've clarified the docstrings.

Sadly, `PhotometryModel` and `AstrometryModel` don't actually share a parent class, though they probably should (so that this API can be defined up there). `AstrometryModel.validate()` exists to fulfill that API, but I need to think through what it should be in practice. I've filed a ticket for the implementation: it would almost certainly be implemented in `AstrometryModel` itself, not in a child class.

Reply:
Oh wait, `validate` can't be overridden. I've been working in Python too long. 🙄